Library/Formula/subversion.rb

```diff
@@ -100,7 +96,7 @@ def install
                 CFLAGS=#{ENV.cflags} LINKFLAGS=#{ENV.ldflags}
                 OPENSSL=#{Formula["openssl"].opt_prefix}]
 
-      unless MacOS::CLT.installed?
+      if build.with?("apr") || !MacOS::CLT.installed?
```

Comment:
This will never get triggered on systems without the CLT installed. I killed off my own CLT and:

```
scons: Reading SConscript files ...
/bin/sh: /usr/local/opt/apr: No such file or directory
OSError: '/usr/local/opt/apr --cflags --cppflags --ldflags --includes --link-ld --libs' exited 127:
  File "/private/tmp/subversion--serf20150810-80964-6wliba/serf-1.3.8/SConstruct", line 341:
    env.ParseConfig('$APR --cflags --cppflags --ldflags --includes'
  File "/usr/local/Cellar/scons/2.3.6/libexec/scons-local/SCons/Environment.py", line 1582:
    return function(self, self.backtick(command))
  File "/usr/local/Cellar/scons/2.3.6/libexec/scons-local/SCons/Environment.py", line 597:
    raise OSError("'%s' exited %d" % (command, status))
```

The current dependency resolution works because it knows when to enforce the fallback, and that happens by default. For users to get that behaviour now they'd have to pass `brew install svn --with-apr` which isn't intuitive when that hasn't never been our default behaviour.

We need to keep the `depends_on :apr => :build` line and make it work with the newly necessary flag.